loss.py

```diff
@@ -15,8 +15,16 @@ def __init__(self, E, data=[x,y]):
     def value(self, params):
         return self.formula.subs([(a, params[0]),(b, params[1])])
     
-    def plot(self, range_a, range_b):
-        sym.plotting.plot3d(self.formula, (a, range_a[0],range_a[1]), (b, range_b[0], range_b[1]))
+    def plot(self, range_a, range_b, include_minima=False):
+        p_1 = sym.plotting.plot3d(self.formula, (a, range_a[0],range_a[1]), (b, range_b[0], range_b[1]), show =False)
+        if include_minima == True:
+            p_2 = sym.plotting.plot3d_parametric_line((u,self.global_minima.subs(a,u), 0.0000001*u, (u, 0.2, 5)), (v,self.global_minima.subs(a,v), 0.0000001*v, (v, -5, -0.2)), show =False)  
+            p_2[0].line_color='r'   
+            p_2[1].line_color='r' 
+            p_1.extend(p_2)
+
+        p_1.show()
+

```

Comment:
Changed plot so that it creates a Plot object first, which it doesn't show. It then checks to see if it should add the minima to the plot. After checking it then shows the plot.